### PR TITLE
Insignificant fix mix

### PR DESCRIPTION
**Pull request by jpopelka**

### CODE REVIEW

**Comment by jpopelka** [comment]
recheck

File: packit_service/worker/handlers/abstract.py [hunk 1]
```diff
@@ -174,7 +174,7 @@ class TaskName(str, enum.Enum):
     installation = "task.run_installation_handler"
     testing_farm = "task.run_testing_farm_handler"
     testing_farm_results = "task.run_testing_farm_results_handler"
-    propose_downstream_comment = "task.run_propose_downstream_comment_handler"
+    # propose_downstream_comment = "task.run_propose_downstream_comment_handler"
```
Comment (TomasTomecek) [inline packit_service/worker/handlers/abstract.py:177]: please put an inline comment here why it's being commented out

**Review by TomasTomecek — APPROVED** [review]

File: packit_service/worker/handlers/abstract.py (hunk 1 above)
Comment (jpopelka) [reply packit_service/worker/handlers/abstract.py]: I had a better idea, check it out now.

**Comment by jpopelka** [comment]
recheck

**Pull request merged**